Makefile

```diff
@@ -155,6 +155,7 @@ gen-manifests: manifest-templator
 	CONTAINER_PREFIX=$(IMAGE_REGISTRY) \
 	CONTAINER_TAG=$(IMAGE_TAG) \
 	MULTUS_IMAGE=$(MULTUS_IMAGE) \
+	MULTUS_V3_IMAGE=$(MULTUS_V3_IMAGE) \
```

Comment:
Maybe I specified the obvious, but since its another flavor of Multus component one could think it works differently, so I wanted to emphasize it should work the same as any other component.
I can remove it if you think it's redundant.

Reply:
I would prefer, it left me more perplexed than anything :) 
The better place to put this info is on the README, and you already did that.